<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmUpdate</class>
 <widget class="QDialog" name="frmUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>474</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Actualizaciones</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/gr-lida.qrc">
    <normaloff>:/img16/actualizar.png</normaloff>:/img16/actualizar.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="vLayout_main">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayout_label">
     <item>
      <widget class="QLabel" name="lb_update_ico">
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/gr-lida.qrc">:/img16/actualizar.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_update">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Actualizaciones</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="AnimatedTextBrowser" name="txtInfo"/>
     <widget class="QTreeWidget" name="twUpdates">
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Version actual</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Versi√≥n nueva</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wInfo" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>24</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="hLayout_wInfo">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_button">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="hSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/descarga.png</normaloff>:/img16/descarga.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>animatedtextbrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>twUpdates</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
